content/blog/2016/2016-01-07-official-code-of-conduct.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Jenkins Code of Conduct"
+author: rtyler
+---
+
+
+Over the past couple months we have been working on a long overdue
+link:/conduct[Code of Conduct] for the Jenkins project (meeting minutes
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2015/jenkins-meeting.2015-12-09-19.01.html[here]
+and
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2016/jenkins-meeting.2016-01-06-19.01.html[here]).
+Following in the footsteps of other projects like the
+link:https://www.apache.org/foundation/policies/conduct.html[Apache Software
+Foundation], link:https://golang.org/conduct[Go lang] and
+link:http://contributor-covenant.org/[countless others], we have adopted this
+code of conduct to help set guidelines for what behaviors are acceptable or
+unacceptable when acting within the Jenkins community or on behalf of the
+Jenkins project.
+
+
+I would like to extend our gratitude to the authors of
+link:http://contributor-covenant.org/[the Contributor Covenant] who provided us
+with a very good and mostly finished Code of Conduct template. We have
+link:/conduct[adapted the covenant] to meet the unique needs of a multifaceted
+project like Jenkins.
+
+
+The document itself is broken down into three sections, although I encourage
```

Comment:
Don't like this sentence. The "although" subordinate clause is just stuck between two unrelated clauses - "The document itself is broken down into three sections, the sections are" wouldn't make much sense.